### PR TITLE
adding rawread test case

### PR DESCRIPTION
**Pull request by Naresh-ibm**
Adding the basic rawread test script.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[root@ltczz401-lp4 disk]# avocado run rawread.py -m rawread.py.data/rawread.yaml 
JOB ID     : 8bdb0dd9f68a82ca39afc3235374789086efbebd
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-11-27T22.06-8bdb0dd/job.log
 (1/1) rawread.py:Rawread.test;run-82f4: PASS (53.48 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 53.58 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-11-27T22.06-8bdb0dd/results.html
[root@ltczz401-lp4 disk]#

**Comment by Naresh-ibm** [comment]
[job.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/3899851/job.log)

File: io/disk/rawread.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        Run 'rawread' with its arguments
+        """
+        err_val = []
+        for val in range(24):
+            cmd = "./rawread -t %s %s " % (val, self.disk)
+            if process.system(cmd, shell=True, ignore_status=True):
+                err_val.append(val)
+
+        if err_val:
+            self.fail("test failed for values : %s" % " ".join(str(err_val)))
```
Comment (narasimhan-v) [inline io/disk/rawread.py]: @Naresh-ibm What exactly are you trying to achieve by `" ".join(str(err_val))`

Comment (Naresh-ibm) [reply io/disk/rawread.py]: @narasimhan-v changed and did a force push. please review and merge.

File: io/disk/rawread.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        Run 'rawread' with its arguments
+        """
+        err_val = []
+        for val in range(24):
+            cmd = "./rawread -t %s %s " % (val, self.disk)
+            if process.system(cmd, shell=True, ignore_status=True):
+                err_val.append(val)
+
+        if err_val:
+            self.fail("test failed for values : %s" % str(err_val))
```
Comment (narasimhan-v) [inline io/disk/rawread.py]: cmd = "./rawread -t %s %s " % (val, self.disk)
            if process.system(cmd, shell=True, ignore_status=True):
                err_val.append(str(val))

        if err_val:
            self.fail("test failed for values : %s" % " ".join(err_val))
```

**Pull request merged**